src/validation/beneficial.owner.individual.validation.ts

```diff
@@ -1,12 +1,14 @@
 import { body } from "express-validator";
 
 import { ErrorMessages } from "./error.messages";
-import { usual_residential_service_address_max_validations, usual_residential_address_max_validations } from "./fields/address.validation";
+import { usual_residential_service_address_beneficial_owner_validations, usual_residential_address_beneficial_owner_validations } from "./fields/address.validation";
+import { VALID_CHARACTERS } from "./regex/regex.validation";
 
 export const beneficialOwnerIndividual = [
-  body("first_name").isLength({ max: 50 }).withMessage(ErrorMessages.MAX_FIRST_NAME_LENGTH),
-  body("last_name").isLength({ max: 160 }).withMessage(ErrorMessages.MAX_LAST_NAME_LENGTH),
+  body("first_name").isLength({ max: 50 }).withMessage(ErrorMessages.MAX_FIRST_NAME_LENGTH).matches(VALID_CHARACTERS).withMessage(ErrorMessages.FIRST_NAME_INVALID_CHARACTERS),
```

Comment:
This is minor and not strictly related to your change, but I find that indenting the validations a bit like this make them a lot more readable:

body("first_name")
  .isLength({ max: 50 }).withMessage(ErrorMessages.MAX_FIRST_NAME_LENGTH)
.matches(VALID_CHARACTERS).withMessage(ErrorMessages.FIRST_NAME_INVALID_CHARACTERS),

Formatting a bit odd in the GitHub comment window, but basically putting each validation check/message pair on a separate indented line. What do you think?